manage-data/lifecycle/index-lifecycle-management/manage-lifecycle-integrations-data.md

```diff
@@ -0,0 +1,64 @@
+---
+navigation_title: Manage the lifecycle for integrations data
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# Manage the lifecycle for integrations data [ilm-manage-data-from-integrations]
```

Comment:
I was thinking about the title and how to make it a bit more specific (without mentioning ILM). Very tricky one. Maybe something like **Managing lifecycle policies for your integrations data** or **Managing lifecycle settings for your integrations data**? (But the settings are defined in a policy so it's kind of the same thing, isn' it?)

If I was to provide even more detail, I'd probably go with
**Managing the lifecycle of data streams generated by integrations**, but that might be a bit long for the nav title, but specific enough for the topic title.

Not sure if any of these suggestions make sense or reflect the content well, but that's what it seems like to me (who doesn't know enough about it).